src/views/BreadView/BreadTable.js

```diff
@@ -0,0 +1,178 @@
+          <TableCell className={classes.buttonCell}>
+            <IconButton title="Lock" disabled>
+              <LockIcon fontSize="small" />
+            </IconButton>
+          </TableCell>
+        </TableRow>
+    )
+};
+
+
+class BreadTable extends React.Component {
+
+  state = {
+    dialogOpen: false,
+    submitting: false,
+    order: {},
+    items: {},
+    selectedItem: "buh"
+  }
+
+  handleAddClick = (order) => () => {
+    this.setState({dialogOpen: true, order: order});
+  }
+
+  handleClearOrder = (order) => () => {
+      this.setState(
+          {dialogOpen: false, order: order},
+          () => this.doClear()
+      );
+  }
+
+  doClear = () => {
+    api.del({
+        path: '/bread/' + this.state.order.id,
+    }).then(data => {
+        this.setState(
+          {submitting: false, order: {}},
+          () => this.props.showMessage('Bestelling verwijderd', this.props.refresh)
+        );
+    }).catch(error => {
+        this.setState(
+          {submitting: false, order: {}},
+          () => this.props.showMessage(error.message)
+        );
+    })
+  }
+
+  handleAddToOrder = value => {
+    this.setState(
+        {dialogOpen: false, submitting: true},
+        () => this.doAddItem(value)
+    );
+  }
+
+  doAddItem = value => {
+    api.patch({
+        path: '/bread/' + this.state.order.id,
+        data: { items: [value] },
+    }).then(data => {
+        this.setState(
+          {submitting: false, order: {}},
+          () => this.props.showMessage('Brood toegevoegd', this.props.refresh)
+        );
+    }).catch(error => {
+        this.setState(
+          {submitting: false, order: {}},
+          () => this.props.showMessage(error.message)
+        );
+    })
+  }
+
+  handleDialogChange = (dialogOpen) => () => {
+    this.setState({dialogOpen: dialogOpen});
+  }
+
+  render () {
+    const { classes, orders, loading, items } = this.props;
+
+    return (
+        <React.Fragment>
+          <Table>
+            <TableHead>
+              <TableRow>
+                <TableCell className={classes.dateCell}>Week</TableCell>
+                <TableCell className={classes.ordersCell}>Bestelling</TableCell>
+                <TableCell className={classes.buttonCell}></TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {loading ? emptyRow(classes) : null}
+              {!loading && orders.length === 0 ? emptyRow(classes, 'Geen bestellingen mogelijk') : null}
+              {orders.map(row => {
+                 return (
+                   <TableRow key={row.id}>
+                     <TableCell className={classes.dateCell}>
+                       {utils.formatDate(row.date)}
+                     </TableCell>
+                     <TableCell className={classes.ordersCell}>
+                         {row.items.map(item => item.name).join(", ")}
+                         {/*  {row.items.map(item => item.price/100).reduce((a,b) => a + b, 0)} */}
```

Comment:
zet de prijs mss in een aparte tableCell

Reply:
Ik weet niet of ik de prijs daar wil zetten eigenlijk, niemand heeft daar echt iets aan want de rekening wordt toch apart gepost

Reply:
ik zou dat wel graag van mezelf weten hoeveel de order nu juist gekost heeft ... dan kan je dat zelf nog eens narekenen of op het einde de rekening wel klopt
ik vind het zelf gewoon transparanter naar de user toe ... (ook voor 'online verkopen' moet je transparant zijn)

Reply:
en gewoon het totaal beneden aan de tabel?